src/milia/api/organization.cljc

```diff
@@ -22,8 +22,9 @@
 
 (defn create [data]
   (let [url (make-url "orgs")]
-    (parse-http :post url :http-options {:form-params data})))
-
+    (parse-http :post url :http-options {:form-params data
+                                         :suppress-40x-exceptions? true
```

Comment:
also these should not passed as kvs in http-options, they should be additional args to parse-http